lib/src/r_tree/node.dart

```diff
@@ -110,7 +110,7 @@ abstract class Node<E> implements RTreeContributor {
     }
 
     _minimumBoundingRect = newBoundingRect;
-    return;
+    return _minimumBoundingRect!;
```

Comment:
I'd return `newBoundingRect` because we know it's not null and can avoid the null assertion operator.
```suggestion
    return newBoundingRect;
```

Reply:
Well nevermind then